src/components/Tabs/Tabs.tsx

```diff
@@ -1 +1,63 @@
-export const Tabs = () => {};
+import React from 'react';
+import classNames from 'classnames';
+
+export interface Tab {
+  id: string;
+  title: string;
+  content: string;
+}
+
+type Props = {
+  tabs: Tab[];
+  selectedTabId: string;
+  onTabSelected: (tab: Tab) => void;
+};
+
+export const Tabs: React.FC<Props> = ({
+  tabs,
+  selectedTabId,
+  onTabSelected,
+}) => {
+  const selectedTab = tabs.find(tab => tab.id === selectedTabId) || tabs[0];
+
+  return (
+    <div data-cy="TabsComponent">
+      <div className="tabs is-boxed is-right">
+        <ul>
+          {tabs.map(tab => {
+            const { id, title } = tab;
+
+            return (
+              <li
+                className={classNames(
+                  { 'is-active': id === selectedTabId },
```

Comment:
```suggestion
                  { 'is-active': id === selectedTab.id },
```
![Capture](https://user-images.githubusercontent.com/108527147/197544902-d6a4304c-664b-4d8f-aa4a-08707abc2e55.JPG)